defaults/main.yml

```diff
@@ -43,7 +43,7 @@ alertmanager_wechat_secret: ''
 alertmanager_wechat_corp_id: ''
 
 # First read: https://github.com/prometheus/alertmanager#high-availability
-alertmanager_cluster: {}
+alertmanager_cluster: {listen-address: ""}
```

Comment:
Could you change it to:
```
alertmanager_cluster:
  listen-address: ""
```

so we have the same syntax everywhere?

Reply:
I did it that way so that it fit into README.md nicely/consistently. Does that convince you to have it this way?  I'm also happy to change as you ask - but changing README.md is trickier...

Reply:
Use short syntax (`alertmanager_cluster: {listen-address: ""}`) in readme and more verbose one in `defaults/main.yml`